content/docs/intro/languages/dotnet.md

```diff
@@ -0,0 +1,94 @@
+---
+title: ".NET"
+menu:
+  intro:
+    parent: languages
+    weight: 2
+
+aliases: ["/docs/reference/dotnet/"]
```

Comment:
The alias to `/docs/reference/dotnet/` should be removed. Aliases are used to create redirects, which we often use when we've moved a page from an old location to a new location. That way, if anyone tries to visit the old URL, they'll be redirected to the new location. Since we haven't previously published anything at `/docs/reference/dotnet/`, we don't need this redirect.

However, you mentioned we do want https://pulumi.com/dotnet to redirect to this landing page. We can use an alias to do that:

```suggestion
aliases: ["/dotnet/"]
```

With that, any visits to https://pulumi.com/dotnet will be redirected to https://pulumi.com/docs/intro/languages/dotnet/

Reply:
fixed